akka-javasdk/src/main/resources/reference.conf

```diff
@@ -53,12 +53,14 @@ akka.javasdk {
   }
 
   entity {
-    # When a EventSourcedEntity or KeyValueEntity is deleted the existence of the entity is completely cleaned up after
+    # When a EventSourcedEntity, KeyValueEntity or Workflow is deleted the existence of the entity is completely cleaned up after
```

Comment:
Is that a public api? hmm maybe we should rename this config to `stateful-component.cleanup-deleted-after` ?

Reply:
ok, makes sense